content/en/blog/2021/discovery-selectors/index.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "Use DiscoverySelectors To Configure Namespaces For Your Istio Service Mesh"
+description: Learn how to use DiscoverySelectors and how it intersects with Sidecar resources.
+publishdate: 2021-04-30
+attribution: "Lin Sun (Solo.io), Christian Posta (Solo.io), Harvey Xia (Solo.io)"
+keywords: [discoveryselectors,Istio,namespaces,sidecar]
+---
+
+As users move their services to run in a service mesh, they are often surprised that the Istio control plane watches and processes all of the Istio and Kubernetes resources from all namespaces in the cluster by default. This can be an issue for very large clusters with lots of namespaces and deployments or even for a moderately sized cluster with rapidly churning resources (for example, Spark jobs).
```

Comment:
```suggestion
As users move their services to run in the Istio service mesh, they are often surprised that the control plane watches and processes all of the Kubernetes resources, from all namespaces in the cluster, by default. This can be an issue for very large clusters with lots of namespaces and deployments, or even for a moderately sized cluster with rapidly churning resources (for example, Spark jobs).
```